### ISSUE TITLE
New router, Can't get through proxy.

### ISSUE THREAD

**Issue by lucia9096** [kakaroto/SWProxy#108]
So i change to a new router and now i tried to use the swproxy and my SW on my phone isn't able to connect. Does anyone know how to fix this on a step by step guide?

**Comment by msteudel** [comment]
Probably just need to restart SWProxy and use the new ip address.

**Comment by lucia9096** [comment]
Ive tried both of those and its still not getting a response. I think the problem is with the router. Is there something that i am missing here?

**Comment by msteudel** [comment]
Can you surf the internet after you've entered your proxy information?

**Comment by lucia9096** [comment]
Yeah i can surf the net though. All im receiving from SW is that the "Failed to connect with the network." message appear at the start of the loading screen.

**Comment by kakaroto** [comment]
in that case, it works. Could it be unrelated to the new router but instead related to the new version of SW that was recently released ? in which case, I think others have already figured out how to fix it (I haven't been following the development here in a while)

**Comment by lucia9096** [comment]
Hello Lord Kakaroto. I think there was a new version of SW. I got friends having the same problems as me recently as well.
